packages/shorebird_cli/lib/src/executables/gradlew.dart

```diff
@@ -113,3 +117,16 @@ class Gradlew {
     return productFlavors;
   }
 }
+
+extension on String {
+  bool get isAcronymPrefixed {
```

Comment:
What is meant by "acronym" here? I worry this isn't as robust as we might want. For example, what behavior would I expect for the following flavors?

- `MyCoolFlavor`
- `NY`
- `NYC`
- `HAL9000`
- `HALNineThousand`

Some tests for this specifically would probably be good.

Reply:
Also: docs

Reply:
> acronym as in the literal meaning of the word, the combination of the first letters of different words.

That's what's confusing me. "mRNA" and "laser" are both acronyms, for example. I think something more to the point ("startsWithUpperCaseLetters" or something) would be clearer.